src/compiler/checker.ts

```diff
@@ -7095,13 +7095,18 @@ namespace ts {
 
                 // Look up the value in the current scope
                 if (node.tagName.kind === SyntaxKind.Identifier) {
-                    valueSymbol = getResolvedSymbol(<Identifier>node.tagName);

```

Comment:
Gaah, symbol duality. You could try `getResolvedSymbol(<Identifier>node.tagName).valueDeclaration.localSymbol`